test/cookie.js

```diff
@@ -37,6 +37,14 @@ describe('new Cookie()', function () {
       assert.strictEqual(typeof new Cookie({}), 'object')
     })
 
+    it('should create cookie object with data field in options', function () {
+      assert.doesNotThrow(function () {
+        var cookie = new Cookie({
+          data: {}
```

Comment:
Why would your code be passing in an option that is simply discarded? If this is what is causing the error, ideally you should remove the part from your code that is passing in this option.

Reply:
Yes, it's true. But function shouldn't break if it accepts some extra data. I think your next comment is good.

Reply:
Right, but the setting controls more than just passing in more data to the function. For example it now enables `req.session.data = 'foo'` which would have been an error even in 1.15.x.